docs/msbuild/msbuild-items.md

```diff
@@ -330,6 +330,259 @@ Output:
 -->
 ```
 
+##  <a name="BKMK_UpdatingMetadataOnExistingItemsOutsideTargets"></a> Updating metadata on items in an ItemGroup outside of a Target
```

Comment:
We generate anchor links automatically now - anchor will be #updating-metadata-on-items-in-an-itemgroup-outside-of-a-target .
```suggestion
##  Updating metadata on items in an ItemGroup outside of a Target
```